include/RE/M/MenuTopicManager.h

```diff
@@ -6,6 +6,7 @@
 #include "RE/B/BSTEvent.h"
 #include "RE/B/BSTList.h"
 #include "RE/B/BSTSingleton.h"
+#include "RE/D/DialogueItem.h"
```

Comment:
```suggestion
```
Is this needed?

Reply:
if i avoid creating a cpp file its not but if a cpp file is used then yes

Reply:
essentially DialogueResponse becomes undefined